packages/components/breadcrumbs/breadcrumbs.ts

```diff
@@ -0,0 +1,161 @@
+import { AsyncPipe, NgTemplateOutlet } from '@angular/common';
+import {
+    AfterContentInit,
+    booleanAttribute,
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    ContentChild,
+    ContentChildren,
+    DestroyRef,
+    Directive,
+    inject,
+    InjectionToken,
+    Input,
+    OnInit,
+    Provider,
+    QueryList,
+    TemplateRef,
+    ViewEncapsulation
+} from '@angular/core';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+import { RouterLink } from '@angular/router';
+import { KbqButton, KbqButtonModule, KbqButtonStyles } from '@koobiq/components/button';
+import { KbqComponentColors, KbqDefaultSizes } from '@koobiq/components/core';
+import { KbqDropdownModule } from '@koobiq/components/dropdown';
+import { KbqIconModule } from '@koobiq/components/icon';
+import { RdxRovingFocusGroupDirective, RdxRovingFocusItemDirective } from '@radix-ng/primitives/roving-focus';
+
+export interface KbqBreadcrumbsConfiguration {
+    max: number | null;
+    size: KbqDefaultSizes;
+    separator?: string;
+}
+
+export const KBQ_BREADCRUMBS_DEFAULT_CONFIGURATION: KbqBreadcrumbsConfiguration = {
```

Comment:
кажется что экспорт не нужен
```suggestion
const KBQ_BREADCRUMBS_DEFAULT_CONFIGURATION: KbqBreadcrumbsConfiguration = {
```